bin/rspecq

```diff
@@ -100,19 +106,21 @@ if opts[:report]
     build_id: opts[:build],
     timeout: opts[:report_timeout],
     redis_host: opts[:redis_host],
+    fail_fast: opts[:fail_fast]
```

Comment:
Likewise, let's pass this as an `attr_accessor` instead of making it a required argument.